src/app/components/work-item-detail/work-item-detail.component.ts

```diff
@@ -897,6 +828,8 @@ export class WorkItemDetailComponent implements OnInit, AfterViewInit, OnDestroy
   }
 
   iterationUpdated(iterationId: string): void {
+    if (iterationId === '0') return; // Loading item
```

Comment:
We need to emit the emitEditWI event in the iterationUpdated function as save function is no longer emitting it.